testing/sdk_install.py

```diff
@@ -22,31 +22,36 @@ def install(
         service_name = package_name
     start = time.time()
     merged_options = get_package_options(additional_options)
+
     print('Installing {} with options={} version={}'.format(
         package_name, merged_options, package_version))
+
     # install_package_and_wait silently waits for all marathon deployments to clear.
     # to give some visibility, install in the following order:
     # 1. install package
     shakedown.install_package(
         package_name,
         package_version=package_version,
         options_json=merged_options)
+
     # 2. wait for expected tasks to come up
     sdk_tasks.check_running(service_name, running_task_count)
+
     # 3. check service health
     app_id = sdk_marathon.get_config(service_name)['id']
     marathon_client = dcos.marathon.create_client()
     def fn():
         # TODO(nickbp): upstream fix to shakedown, which currently checks for ANY deployments rather
         #               than the one we care about
         deploying_apps = set([])
-        for d in marathon_client.get_deployments():
-            for a in d.get('affectedApps', []):
-                deploying_apps.add(a)
+        for deployment in marathon_client.get_deployments():
```

Comment:
Was the intention to also add a check that a framework is suppressed once install is complete?